docker/hooks/post_push

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright 2019 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of iotagent-json
+#
+# iotagent-json is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# iotagent-json is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with iotagent-json. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact
+# with iot_support at tid dot es
+
+docker image push $IMAGE_NAME-pm2 
+
+docker image push $IMAGE_NAME-distroless
```

Comment:
I understand that the default image (the one for node-slim) is automatically pushed so there is no need of including it in posst_push. Is my understanding correct?

Reply:
That's correct @fgalan. The tagged version is pushed by default. So there is no need to push it again.

Reply:
NTC